packages/ui-causal-graph-editor/src/types.tsx

```diff
@@ -203,11 +203,10 @@ export type SimulationGraph = AbstractGraph<SimulationNode, SimulationEdge, Simu
  * Int values from cl_interfaces.EdgeConstraint
  */
 export enum EdgeConstraintType {
```

Comment:
Is the comment above the still accurate or did the definition move? It's somewhere in `cai-causal-graph` I think?

Reply:
good catch, it has both changed places and type, fixed